.github/workflows/ci.yml

```diff
@@ -54,8 +54,9 @@ jobs:
 
       - name: Install required packages
         run: |
-          pip install -r runtime_k3d/src/requirements.txt
-          pip install -r runtime_k3d/test/requirements.txt
+          pip install -r desired_state_generator/src/requirements.txt
```

Comment:
i will just place the comment here. its regarding some lines up:

container: ghcr.io/eclipse-velocitas/devcontainer-base-images/python:v0.1

could you update to v0.2 or even already v0.3 (later when the base image is released)

Reply:
done